src/mono/sample/mbr/DeltaHelper/DeltaHelper.cs

```diff
@@ -48,7 +48,8 @@ public class DeltaHelper {
 
         private static void LoadMetadataUpdate (Assembly assm, byte[] dmeta_data, byte[] dil_data, byte[] dpdb_data)
         {
-            UpdateMethod (assm, dmeta_data, dil_data, dpdb_data);
+            // UpdateMethod (assm, dmeta_data, dil_data, dpdb_data);
+	    System.Reflection.Metadata.AssemblyExtensions.ApplyUpdate (assm, dmeta_data, dil_data, dpdb_data);
```

Comment:
Indentation looks wrong here, are you mixing tabs and spaces?

Reply:
bah. yes. and also I meant to delete a bunch of the old code in here.